test/test_problems.py

```diff
@@ -215,7 +215,7 @@ def test_new_contest_problem(self, monkeypatch):
 
 class TestReadProblemConfig:
     def test_read_problem_config(self):
-        p = problem.Problem(RUN_DIR / "test/problems/test_problem_config", Path("/tmp/xyz"))
+        p = problem.Problem(RUN_DIR / "test/problems/testProblemConfig", Path("/tmp/xyz"))
```

Comment:
I'm guessing your removed the underscores to follow the spec, but capital letters are also not allowed:

> The directory name must consist solely of lowercase letters a–z and digits 0–9.

Reply:
yea... didn't check the warning properly XD